src/commcare_cloud/terraform/modules/logshipping/main.tf

```
@@ -14,3 +16,13 @@ resource "aws_s3_bucket" "log_bucket" {
     }
   }
 }
+
+module "firehose_stream" {
```

I might change the name here to

```suggestion
module "formplayer_request_response_logs_firehose_stream" {
```

---

Same as above, I know we talked about eventually pulling more logs into s3. Are they going to have a separate logshipping "instance" and if so, should we be making it generic?